### DISCUSSION TITLE
Is there a way to automatically apply changes that are saved to a preset to all existing instances?

### DISCUSSION THREAD

**Discussion by ironirc** [helgoboss/helgobox#1309] [Q&A]
I read in section "Reload all compartment presets from disk"
If you want to apply a preset that has been changed on disk, you need to reload it by selecting it in the preset dropdown once again!  

So basically what I'm asking is to automate this step on all instances.

**Comment by helgoboss** [comment]
There's no such option. Presets are more like templates in this regard. It would be dangerous to apply preset changes automatically. 

That said, it would be technically possible to add a button which applies the updated preset (instead having to reselect it). I think it would be best to apply it unit per unit manually. Do you really need this for all instances/units? What's the use case?

**Reply by ironirc** [reply]
Use case: recalling snapshots to FX or containers triggered by program changes. I have duplicate instances spread across multiple channel strips to achieve similar functionality as found on many multitimbral hardware sound modules (e.g., Roland Fantom). The repertoire of presets changes a lot...

**Reply by helgoboss** [reply]
Ah. But then it would be better to introduce a new mechanism for global FX snapshots (vs. just FX snapshots that are local to the mapping)... something more akin Pot Browser. Because multiple FX snapshots together can become quite big and duplicating all of them into multiple instances would make this issue even worse.
